lib/gateway/services/grafana.conf.tpl

```diff
@@ -0,0 +1,12 @@
+location /api/grafana { return 302 https://${BENTOV2_PORTAL_DOMAIN}/api/grafana/; }
+location /api/grafana/ {
+    # Reverse proxy settings
+    include /gateway/conf/proxy.conf;
+    include /gateway/conf/proxy_extra.conf;
+    
+    # Immediate set/re-use means we don't get resolve errors if not up (as opposed to passing as a literal)
+    set $upstream_grafana http://bentov2-grafana:3000;
```

Comment:
instead of hardcoding the upstream name, use the `BENTO_GRAFANA_CONTAINER_NAME` environment variable